### ISSUE TITLE
Episodes not playing 

### ISSUE THREAD

**Issue by Djtwister** [pystardust/ani-cli#928]
**Metadata (please complete the following information)**
Version: [3.4.5]
OS: [Linux Zorin os 16.2]
Shell: [bash 5.0.17]

When trying to play any episode from any anime, it just stays there but nothing happens. I've already installed both mpv and vlc but none of them seem to work. I've also tried to uninstall and install again but it won't work :/

**Steps To Reproduce**

1. Run `cd ani-cli && ./ani-cli`
2. Search Spy x family  
3. Choose 3 
4. Choose episode 5 

**Expected behavior**
Open mpv and reproduce the episode

**Comment by Derisis13** [comment]
Couldn't reproduce on Pop!_OS. Are you getting any error or are you just landed in the main menu (previous, replay, next, select, quit)?

My guess is there's something wrong with mpv/vlc. What versions of mpv/vlc do you have? Some mpv/vlc instances don't include ffmpeg for licensing reasons. Run `mpv --version` to see information about what is included in your installation.

**Comment by HarshSinha11** [comment]
Experiencing same issue nothing happens with mpv. But the Episode plays with VLC. 

I think parameter in the code does not work with my installed mpv 
```nohup "$player_fn" "$video_url" --no-stdin --keep-running --mpv-referrer="$refr" --mpv-force-media-title="$trackma_title" ```
```Error parsing option no-stdin (option not found)```

**Comment by Djtwister** [comment]
> Couldn't reproduce on Pop!_OS. Are you getting any error or are you just landed in the main menu (previous, replay, next, select, quit)?
> 
> My guess is there's something wrong with mpv/vlc. What versions of mpv/vlc do you have? Some mpv/vlc instances don't include ffmpeg for licensing reasons. Run `mpv --version` to see information about what is included in your installation.

It just showed the main menu without playing the episode. I have since hopped to Fedora and the error is no longer present

**Comment by port19x** [comment]
Please someone provide a concrete mpv version

> Experiencing same issue, nothing happens with mpv. But the Episode does play with VLC.
> 
> > My guess is there's something wrong with mpv/vlc.
> 
> I think parameter in the code does not work with my installed mpv `nohup "$player_fn" "$video_url" --no-stdin --keep-running --mpv-referrer="$refr" --mpv-force-media-title="$trackma_title" ` `Error parsing option no-stdin (option not found)`

**Comment by Suxton** [comment]
Happens to me too, mpv-git output: 
```
mpv 0.35.0-38-gb5373079f2 Copyright © 2000-2022 mpv/MPlayer/mplayer2 projects
  built on Sun Dec 18 11:48:46 2022
libplacebo version: v4.208.0
FFmpeg version: n5.1.2
FFmpeg library versions:
   libavutil       57.28.100
   libavcodec      59.37.100
   libavformat     59.27.100
   libswscale      6.7.100
   libavfilter     8.44.100
   libswresample   4.7.100```

**Comment by Derisis13** [comment]
Thank you, as I said, I'd like to see your mpv version as well. To get it run: `mpv --version` and paste the output here.

**Comment by port19x** [comment]
Since we're changing providers and making sweeping changes to ani-cli to fix #948 I'll assume this will be fixed on accident.
If it doesn't you can reopen this in a few days once v4 is out
